scheduler/views.py

```diff
@@ -197,7 +203,18 @@ def edit(request: HttpRequest, lesson_id) -> HttpResponse:
             lesson.start_time = form.cleaned_data['start_time']
             lesson.end_time = form.cleaned_data['end_time']
             lesson.save()
+            db_conflicts()
             context = get_full_context_with_date(form.cleaned_data['start_time'])
+            current_conflicts = list(context['conflicts'])
+            print("was ", len(past_conflicts))
+            print("is ", len(current_conflicts))
+            new_conflicts, removed_conflicts = conflicts_diff(past_conflicts, current_conflicts)
```

Comment:
Te dwie zmienne nie są potem nigdzie wykorzystywane. Jak rozumiem to tylko baza do różnicy w konfliktach robiona na innym branchu?